lib/helpers.py

```diff
@@ -246,6 +245,20 @@ def _drop_duplicates_in_attributions(df, max_timedelta):
 
         return filtered[['ts', 'user_id', 'revenue_eur']]
 
+    def run_export_user_ids(self, marks_and_spend_df):
```

Comment:
No need to add `run_` prefix to the method name: all of them would require one otherwise. It would be better, if it was removed.

Reply:
I removed this method at all